frontend/benefit/applicant/src/components/applications/forms/application/step1/companyInfo/CompanyInfo.tsx

```diff
@@ -0,0 +1,179 @@
+import { APPLICATION_FIELDS } from 'benefit/applicant/constants';
+import { TextInput } from 'hds-react';
+import React from 'react';
+import InputMask from 'react-input-mask';
+import Skeleton from 'react-loading-skeleton';
+import { StyledCheckbox } from 'shared/components/forms/fields/styled';
+import FormSection from 'shared/components/forms/section/FormSection';
+
+import { useComponent } from '../extended';
+import SC from './CompanyInfo.sc';
+import useCompanyInfo from './useCompanyInfo';
+
+const CompanyInfo: React.FC = () => {
+  const { getErrorMessage, fields, translationsBase, formik } = useComponent();
+  const { t, data, isLoading, shouldShowSkeleton, error } = useCompanyInfo();
+
+  return (
+    <FormSection header={t(`${translationsBase}.heading1`)} loading={isLoading}>
+      <SC.CompanyInfoContainer>
+        <SC.CompanyInfoSection>
+          <SC.CompanyInfoColumn>
+            {shouldShowSkeleton ? (
```

Comment:
i think we can replace with one skeleton <Skeleton count={2} />, it does not make much difference if we show 2 lines or 2 lines with 2 columns, but will reduce duplicate code a lot.